services/filesstore/mocks/FileBackend.go

```diff
@@ -73,16 +73,37 @@ func (_m *FileBackend) FileExists(path string) (bool, error) {
 	return r0, r1
 }
 
+// FileSize provides a mock function with given fields: path
+func (_m *FileBackend) FileSize(path string) (int64, error) {
```

Comment:
Is this related?

Reply:
This got generated from `make filestore-mocks`. Since we don't validate this in CI, this gets outdated from time to time.